index.html

```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
              <div class="ellipse8"></div>
              <div class="ellipse15"></div>
          </div>
+         <div class="flex-botu">
+            <div class="resume">Get my resume</div>
+         </div>
+         <ul class="border-lang">
+             <li>Langguages</li>
+         </ul>    
+         <ul class="lang-img">
+             <li><a href="#" class="js-icon"><img src="source/js-icon.png" alt="js-icon"><span>JavaSript</span></a></li>
+             <li><a href="#" class="html-icon"><img src="source/html-icon.png" alt="html-icon"><span>HTML</span></a></li>
+             <li><a href="#" class="css-icon"><img src="source/css-icon.png" alt="css-icon"><span>CSS</span></a></li> 
+         </ul>        
+         <ul class="fr-work">
+             <li class="border-frem">Frameworks</li>
+             <li class="border-skill">Skills</li>
+         </ul>
+         <div class="work-sec">
+             <div class="snapshoot-img"></div>
+             <ul class="tonic">
+                 <li>Tonic</li>
```

Comment:
- [ ] As requested by the previous code reviewer, please make the project titles (e.g. Tonic) are surrounded by an `<hx>` tags.